### PR TITLE
Pyrewood Village Event Rework

### PR DESCRIPTION
**Pull request by Oroxzy**
**Facts (Classic):**

1. Start is 21:00, End is 05:30.
2. Quest 452 (Pyrewood Ambush) is not affected.
3. 48 NPC's are affected.
4. Apothecary Berard (2106) is not affected and is always a Worgen.
5. Worgen have different Spells.
6. Worgen have no Weapon slots.

### CODE REVIEW

File: sql/migrations/20220107120001_world.sql [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220107120001');
+-- Add your query below.
+
+
+SET @PYREWOOD_EVENT = 49;
+SET @PYREWOOD_VILLAGE_CGUID = 57441;
+
+-- New Event for Pyrewood.
+INSERT INTO `game_event` (`entry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `hardcoded`, `disabled`, `patch_min`, `patch_max`) VALUES (@PYREWOOD_EVENT, '2020-01-02 21:00:00', '2030-12-31 08:00:00', 1440, 720, 0, 'Pyrewood Village Night', 0, 0, 0, 10);
```
Comment (scottnice) [inline sql/migrations/20220107120001_world.sql:16]: Should this use the existing 'Night' event?

File: sql/migrations/20220107120001_world.sql [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220107120001');
+-- Add your query below.
+
+
+SET @PYREWOOD_EVENT = 49;
+SET @PYREWOOD_VILLAGE_CGUID = 57441;
+
+-- New Event for Pyrewood starts 21:00, ends 05:30 (confirmed by ptr).
+INSERT INTO `game_event` (`entry`, `start_time`, `end_time`, `occurence`, `length`, `holiday`, `description`, `hardcoded`, `disabled`, `patch_min`, `patch_max`) VALUES (@PYREWOOD_EVENT, '2020-01-02 21:00:00', '2030-12-31 05:30:00', 1440, 720, 0, 'Pyrewood Village Night', 0, 0, 0, 10);
```
Comment (scottnice) [inline sql/migrations/20220107120001_world.sql]: do you need to update length to correspond with 5:30am? Also should the silverpine script be updated to 5:30am for the pyrewood ambush transforms?

Comment (Oroxzy) [reply sql/migrations/20220107120001_world.sql]: i've already done everything incl. moved entire ambush quest to db. but not pushed yet.

Comment (scottnice) [reply sql/migrations/20220107120001_world.sql]: what about the length of 12 hours?

**Pull request merged**